pkg/fakerp/dns.go

```diff
@@ -123,15 +125,19 @@ func DeleteOCPDNS(ctx context.Context, subscriptionID, resourceGroup, dnsResourc
 	zc.Authorizer = authorizer
 	rsc := dns.NewRecordSetsClient(subscriptionID)
 	rsc.Authorizer = authorizer
+	count := new(int32)
```

Comment:
@mjudeikis @jim-minter I wasn't sure how to create a pointer for this.  A few other iterations resulted in nil pointer exceptions. I picked this up from SO.

Any suggestions?

Reply:
you can use `to.Int32Ptr()` from the `github.com/Azure/go-autorest/autorest/to` package

Reply:
Let me try this.